content-src/components/DiscoveryStreamComponents/Hero/_Hero.scss

```diff
@@ -14,11 +14,12 @@ $card-header-in-hero-line-height: 20;
   }
 
   .excerpt {
-    @include limit-visibile-lines(3, 20, 14);
     @include dark-theme-only {
       color: $grey-10;
     }
 
+    line-height: 20px;
+    font-size: 14px;
```

Comment:
Should these be using `$card-header-in-hero-line-height` ? The variables at the top seem to be unused.